Content/NPCs/Forest/Blover.cs

```diff
@@ -35,6 +35,8 @@ internal class Blover : ModNPC
 
         private bool blowing = false;
 
+        private int blowCounter = 0;
```

Comment:
lets use ref AI fields first before we create custom variables for ease of multiplayer implementation later